pkg/analyze/host_time_test.go

```diff
@@ -200,10 +211,56 @@ func TestAnalyzeHostTime(t *testing.T) {
 					},
 				},
 			},
-			result: &AnalyzeResult{
-				Title:   "Time",
-				IsFail:  true,
-				Message: "Timezone is not set to UTC",
+			result: []*AnalyzeResult{
+				{
+					Title:   "Time",
+					IsFail:  true,
+					Message: "Timezone is not set to UTC",
+				},
+			},
+		},
+		{
+			name: "multiple issues",
+			timeInfo: &collect.TimeInfo{
+				Timezone:        "PST",
+				NTPSynchronized: false,
+				NTPActive:       true,
+			},
+			hostAnalyzer: &troubleshootv1beta2.TimeAnalyze{
+				Outcomes: []*troubleshootv1beta2.Outcome{
+					{
+						Fail: &troubleshootv1beta2.SingleOutcome{
+							When:    "timezone != UTC",
+							Message: "timezone is not set to UTC",
+						},
+						Pass: &troubleshootv1beta2.SingleOutcome{
+							When:    "timezone == UTC",
```

Comment:
each outcome should only have one of pass/fail/warn, not multiple

I think that's in another test in this file, too